bin/ci-test

```diff
@@ -1,17 +1,19 @@
 #!/bin/bash -e
-#/ usage: ci-test SUITE
-#/ simulate the process followed by ~/.github/workflows/ci.yml
+#/ usage: ci-test build-core|build-core-ci|<SUITE>|run-suite <SUITE>
```

Comment:
This usage is kind of difficult to read. I think we can leave more extensive comments

```suggestion
#/ usages: 
#/  - ci-test <SUITES>
#/  - ci-test build-core
#/  - ci-test build-core-ci
#/  - ci-test run-suite <SUITE>
```